src/show/pages/public/Standings/index.jsx

```diff
@@ -1,70 +1,70 @@
-              logo='https://logodownload.org/wp-content/uploads/2014/04/mercedes-benz-logo-1-1.png'
-              place='9'
-              points='176'
-            />
-            <ConstructorStandingCard
-              logo='https://logodownload.org/wp-content/uploads/2014/04/mercedes-benz-logo-1-1.png'
-              place='10'
-              points='176'
-            />
+    <div className='flex w-full flex-col'>
+      <div className='flex w-full  overflow-y-auto flex-col bg-primary'>
+        <div className='flex w-full  bg-white h-52 overflow-x-auto'>
+          <div className='flex ml-5 mt-3 flex-col text-charade text-2xl font-bold antialiased'>
+            Constructors Standing
+            <div className='flex flex-row'>
+              <ConstructorStandingCard constructor={constructor} />
+              <ConstructorStandingCard constructor={constructor} />
+              <ConstructorStandingCard constructor={constructor} />
+              <ConstructorStandingCard constructor={constructor} />
+              <ConstructorStandingCard constructor={constructor} />
+              <ConstructorStandingCard constructor={constructor} />
+              <ConstructorStandingCard constructor={constructor} />
+              <ConstructorStandingCard constructor={constructor} />
+              <ConstructorStandingCard constructor={constructor} />
+              <ConstructorStandingCard constructor={constructor} />
+            </div>
           </div>
         </div>
-      </div>
-      <div className='flex ml-5 mt-3 flex-col text-white text-2xl font-bold antialiased'>
-        Drivers Standing
-        <div className='flex flex-col'>
-          <DriverStandingCard />
+        <div className='flex ml-5 mt-3 flex-col text-white text-2xl font-bold antialiased'>
+          Drivers Standing
+          <div className='flex flex-col'>
+            <div className='flex mt-3 flex-col'>
+              <DriverStandingCard driver={driver} />
+              <DriverStandingCard driver={driver} />
+              <DriverStandingCard driver={driver} />
+              <DriverStandingCard driver={driver} />
+              <DriverStandingCard driver={driver} />
+              <DriverStandingCard driver={driver} />
+              <DriverStandingCard driver={driver} />
+              <DriverStandingCard driver={driver} />
+              <DriverStandingCard driver={driver} />
+              <DriverStandingCard driver={driver} />
+              <DriverStandingCard driver={driver} />
+              <DriverStandingCard driver={driver} />
+              <DriverStandingCard driver={driver} />
+              <DriverStandingCard driver={driver} />
+              <DriverStandingCard driver={driver} />
+              <DriverStandingCard driver={driver} />
+              <DriverStandingCard driver={driver} />
+              <DriverStandingCard driver={driver} />
+              <DriverStandingCard driver={driver} />
+              <DriverStandingCard driver={driver} />
```

Comment:
And an array of drivers and map through each any every one of them to show a <DriverStadingCard />